plugins/pulp_python/plugins/distributors/steps.py

```diff
@@ -121,6 +133,55 @@ def _create_package_index(name, simple_path, packages):
 
             package_index.write(ElementTree.tostring(html, 'utf8'))
 
+    @staticmethod
+    def _create_metadata(name, packages):
```

Comment:
I feel like there should be a model for ``Project``, and the ``Package`` model should have a [many-to-one](https://docs.djangoproject.com/en/1.9/topics/db/models/#many-to-one-relationships) relationship with a ``Project``. This could be an instance method of ``Project``. @seandst, does that sound crazy?